src/main/java/ee/sk/siddemo/model/UserRequest.java

```diff
@@ -30,13 +30,13 @@
 public class UserRequest {
 
     @NotNull
-    @Pattern(regexp="[EE|LV|LT]",
-            message="Invalid country number")
+    @Pattern(regexp = "[EE|LV|LT]",
```

Comment:
Praegu pole oluline, aga kui SK tahab seda demo teenust edasi kasutama jääda, siis peaks need validatsioonid leebemaks tegema (teiste riikide, nt BE, valguses).

Reply:
Mhm, BE võiks tegelikult juba ära lisada.

Reply:
Hetkel need valideerimised ei tööta. Need on lihtsalt näilised.

Reply:
Tegin täienduse ära. Lisasin BE juurde ja hibernate-validatori. Katsetasin ka erinevate riikidega läbi https://github.com/SK-EID/smart-id-documentation/wiki/Environment-technical-parameters#test-accounts-for-automated-testing